block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagRepresentationSyntax.scala

```diff
@@ -48,6 +51,16 @@ final class BlockDagRepresentationOps[F[_]: Sync](
     fs2.Stream.emits(streams).parJoinUnbounded.compile.toList
   }
 
+  def latestMessageHashUnsafe(v: Validator)(
+      implicit line: sourcecode.Line,
+      file: sourcecode.File,
+      enclosing: sourcecode.Enclosing
```

Comment:
With monix 3.0 and sync stack trace support we don't need to manually write source info.